Tests/SpeziFoundationTests/MainActorExecutionTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+
+import SpeziFoundation
+import XCTest
+
+
+@globalActor
+actor TestActor: GlobalActor {
```

Comment:
Might make sense to mark this `fileprivate`, but definitely no big deal

Reply:
good catch 👍